manifest-go115.tmpl

```diff
@@ -11,4 +11,10 @@ manifests:
     platform:
       architecture: amd64
       os: windows
-      version: 20H2
\ No newline at end of file
+      version: 20H2
+  -
+    image: rancher/golang:1.15-windowsservercore-ltsc2022
+    platform:
+      architecture: amd64
+      os: windows
+      version: ltsc2022
```

Comment:
Did you mean to add 2022 support to go1.15? I think we should stick with 1.16 only until we add 1.17 and then have 1.16 and 1.17 be the only versions that have 2022 builds

Reply:
not sure this really hurts anything building this tho